mesos-swarm-marathon/configure-mesos-cluster.sh

```diff
@@ -11,6 +11,7 @@
 ###########################################################
 
 echo "starting mesos cluster configuration"
```

Comment:
Echo statements can fail too. It is a program. I am not saying this script has failure checks. I am saying if something fails in this scrpt, script should exit immediately. Bash default behavior is to continue without exitinf. Please google "bash strict mode" @anhowe